package.json

```diff
@@ -80,10 +80,15 @@
     "vscode-builtin-json-language-features": "https://open-vsx.org/api/vscode/json-language-features/1.46.1/file/vscode.json-language-features-1.46.1.vsix",
     "cortex-debug": "https://open-vsx.org/api/marus25/cortex-debug/0.3.10/file/marus25.cortex-debug-0.3.10.vsix",
     "vscode-language-pack-nl": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-nl/1.48.3/file/MS-CEINTL.vscode-language-pack-nl-1.48.3.vsix",
-    "vscode-language-pack-fr": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-fr/1.53.2/file/MS-CEINTL.vscode-language-pack-fr-1.53.2.vsix",
-    "vscode-language-pack-zh-hans": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-zh-hans/1.53.2/file/MS-CEINTL.vscode-language-pack-zh-hans-1.53.2.vsix",
-    "vscode-language-pack-de": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-de/1.53.2/file/MS-CEINTL.vscode-language-pack-de-1.53.2.vsix",
-    "vscode-language-pack-ja": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-ja/1.53.2/file/MS-CEINTL.vscode-language-pack-ja-1.53.2.vsix",
-    "vscode-language-pack-tr": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-tr/1.53.2/file/MS-CEINTL.vscode-language-pack-tr-1.53.2.vsix"
+    "vscode-language-pack-fr": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-fr/1.69.0/file/MS-CEINTL.vscode-language-pack-fr-1.69.0.vsix",
+    "vscode-language-pack-zh-hans": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-zh-hans/1.69.0/file/MS-CEINTL.vscode-language-pack-zh-hans-1.69.0.vsix",
+    "vscode-language-pack-de": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-de/1.69.0/file/MS-CEINTL.vscode-language-pack-de-1.69.0.vsix",
+    "vscode-language-pack-ja": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-ja/1.69.0/file/MS-CEINTL.vscode-language-pack-ja-1.69.0.vsix",
+    "vscode-language-pack-tr": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-tr/1.69.0/file/MS-CEINTL.vscode-language-pack-tr-1.69.0.vsix",
+    "vscode-language-pack-it": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-it/1.69.0/file/MS-CEINTL.vscode-language-pack-it-1.69.0.vsix",
+    "vscode-language-pack-ru":"https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-ru/1.69.0/file/MS-CEINTL.vscode-language-pack-ru-1.69.0.vsix",
+    "vscode-language-pack-es": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-es/1.69.0/file/MS-CEINTL.vscode-language-pack-es-1.69.0.vsix",
+    "vscode-language-pack-pt-BR": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-pt-BR/1.69.0/file/MS-CEINTL.vscode-language-pack-pt-BR-1.69.0.vsix",
```

Comment:
https://github.com/arduino/arduino-ide/blob/55e83a89d82903b2f21ce47d5fd5d4d3f842480c/arduino-ide-extension/src/node/arduino-localization-contribution.ts#L32-L35

I found that the `locale` argument must match the `contributes.localizations.languageId` value from the language pack in order for the Arduino-specific localizations to take effect. 

This was the only value that did not match with the language packs installed via this PR.

https://github.com/microsoft/vscode-loc/blob/a0cfb4e27f712122066e6f994ef1403e807389bf/i18n/vscode-language-pack-pt-BR/package.json#L22

```json
"languageId": "pt-br",
```

The Arduino IDE after selecting **File > Preferences > Language > português (Brasil)** when using the build from https://github.com/arduino/arduino-ide/commit/55e83a89d82903b2f21ce47d5fd5d4d3f842480c:

![image](https://user-images.githubusercontent.com/8572152/178964677-ede57452-38be-4ddf-a598-3b269ad553d0.png)


The Arduino IDE after changing the the `locale` argument to `pt-br`:

![image](https://user-images.githubusercontent.com/8572152/178965242-222ea997-3c95-4aba-a1e6-7fa317ff48cb.png)

However, I'm wondering about whether it is appropriate to use the `pt-br` language pack with the Arduino Portuguese localization data. I don't see a language pack for `pt` [in the Open VSX Registry](https://open-vsx.org/?search=&category=Language%20Packs).

Reply:
Nice one, thanks. I don't really know what to do here. The reason I chose the Brasilian pack is that it was the only Portuguese pack, but I'm having second thoughts on this choice.

Reply:
I've removed the Portuguese language pack _for now_. I'll re-add it ASAP when we find out how to handle this.

Reply:
After talking about this with our localization specialist, I learned that for now want to avoid splitting dialects (like `pt` and `pt-br`) so as not to disperse the forces of the community. Since the people contributing to the Portuguese translation on Transifex are a mix of Portuguese and Brazilians, we decided to use the `pt-br` language pack.